alignit/config.py

```diff
@@ -149,6 +156,11 @@ class InferConfig:
         default=5,
         metadata={"help": "Number of iterations within tolerance before stopping"},
     )
+    num_multiplications: int = field(
+        default=3,
+        metadata={
+            "help": "Number of times to multiply the transformation matrix"},
```

Comment:
This is not the multiplication of transformation matrices but rotation matrices.

Also, the parameter name and description are too generic.